test_runner/src/main/kotlin/ftl/args/ValidateAndroidArgs.kt

```diff
@@ -58,7 +71,9 @@ private fun AndroidArgs.assertLabelContent() {
 
     if (scenarioNumbers.isNotEmpty() && (type != Type.GAMELOOP))
         throw FlankConfigurationError("Scenario numbers defined but Type is not Game-loop.")
-    scenarioNumbers.forEach { it.toIntOrNull() ?: throw FlankConfigurationError("Invalid scenario number provided - $it") }
+    scenarioNumbers.forEach {
+        it.toIntOrNull() ?: throw FlankConfigurationError("Invalid scenario number provided - $it")
+    }
```

Comment:
```
    scenarioNumbers.validateScenarionsNumbers()
...
fun List<String>.validateScenarionsNumbers() =  forEach {
        it.toIntOrNull() ?: throw FlankConfigurationError("Invalid scenario number provided - $it")
    }
```

Reply:
I dont think this actually achieves anything beyond giving a name to a very simple to read function. WDYT?